### PR TITLE
Improve annotations rendering

### PR DESCRIPTION
**Pull request by vmishenev**
According to  #2015:

1. omit parentheses if there are no parameters 
2. add quotes for string literals into annotations

### CODE REVIEW

File: plugins/base/src/main/kotlin/signatures/JvmSignatureUtils.kt [hunk 1]
```diff
@@ -93,20 +92,29 @@ interface JvmSignatureUtils {
...
+    private fun<T> PageContentBuilder.DocumentableContentBuilder.listParams(
+        params: Collection<T>,
+        listBrackets: Pair<Char, Char>?,
+        outFn: PageContentBuilder.DocumentableContentBuilder.(T) -> Unit
+    ) {
+        listBrackets?.let{ text(it.first.toString()) }
+        params.forEachIndexed { i, it ->
+            group(styles = setOf(TextStyle.BreakableAfter)) {
+                this.outFn(it)
+                if (i != params.size - 1) text(", ")
```
Comment (MarcinAman) [inline plugins/base/src/main/kotlin/signatures/JvmSignatureUtils.kt:112]: you can use `lastIndex`

**Review by MarcinAman — APPROVED** [review]
Could you write a test with `NullValue` being used?

File: plugins/base/src/main/kotlin/signatures/JvmSignatureUtils.kt (hunk 1 above)
Comment (vmishenev) [reply plugins/base/src/main/kotlin/signatures/JvmSignatureUtils.kt:112]: It is only for List or Array, not Collection.

**Comment by vmishenev** [comment]
> Could you write a test with `NullValue` being used?

I already wrote one in  ContentForAnnotationsTest.kt ( `rich documented annotation` ). Need another test for `NullValue`?

**Comment by MarcinAman** [comment]
> > Could you write a test with `NullValue` being used?
> 
> I already wrote one in ContentForAnnotationsTest.kt ( `rich documented annotation` ). Need another test for `NullValue`?

Nah, thats fine. I've missed it ☺️

**Pull request merged**